lib/internal/process/execution.js

```diff
@@ -45,14 +46,24 @@ function evalModule(source, print) {
   const { log } = require('internal/console/global');
   const { loadESM } = require('internal/process/esm_loader');
   const { handleMainPromise } = require('internal/modules/run_main');
-  handleMainPromise(loadESM(async (loader) => {
+  return handleMainPromise(loadESM(async (loader) => {
     const { result } = await loader.eval(source);
     if (print) {
       log(result);
     }
   }));
 }
 
+function evalModuleOrCrash() {
+  const { error } = require('internal/console/global');
+  const { decorateErrorStack } = require('internal/util');
+  FunctionPrototypeApply(evalModule, this, arguments).catch((e) => {
+    decorateErrorStack(e);
+    error(e);
+    process.exit(1);
```

Comment:
@addaleax This catch block was removed in a commit of yours (e948ef351b4111d78ae8ba08f29eb138af22fa0b), should I get rid of it?

Reply:
@aduh95 Yeah, we should always use what `loadESM()` does (i.e. use `triggerUncaughtException()`) when there’s an uncaught exception. `console.error()` + `process.exit()` is not a good solution for this, because it doesn’t handle the case in which these functions throw an exception themselves.